static/app/components/searchSyntax/parser.tsx

```diff
@@ -834,11 +835,12 @@ export class TokenConverter {
     if (
       this.config.validateKeys &&
       this.config.supportedTags &&
-      !this.config.supportedTags[getKeyName(key)]
+      !this.config.supportedTags[getKeyName(key)] &&
+      !this.config.supportedTags[getKeyName(key, {showExplicitTagPrefix: true})]
```

Comment:
Is it possible to send `tags[tag_name, number]` but not `tag_name`? If so this would fail to present an error when you use `tag_name`